chain/messagepool/messagepool.go

```diff
@@ -1584,6 +1614,11 @@ func (mp *MessagePool) Clear(ctx context.Context, local bool) {
 
 			if ok {
 				for _, m := range mset.msgs {
+					if types.BigCmp(co.MinBaseFee, big.Zero()) > 0 &&
+						types.BigCmp(m.Message.GasFeeCap, co.MinBaseFee) < 0 {
```

Comment:
We _don't_ delete messages with `GasFeeCap < MinBaseFee`? Because?

Reply:
This.. maay be in the wrong order

Reply:
That is we should remove messages below the specified minimum basefee

Reply:
That makes more sense.